src/main/java/com/catapult/app/example/services/CallbackServices.java

```diff
@@ -49,17 +45,69 @@ public synchronized void handleIncomingCallback(final String eventString, final
         try {
             Event event = EventBase.createEventFromString(eventString);
             if (event instanceof IncomingCallEvent) {
+                // Answer the incoming call
+                Call incomingCall = answerIncomingCall(event, userName);
 
-                if (event.getProperty("from") != null && sipPattern.matcher(event.getProperty("from")).find()) {
-                    // Case Incoming call from Endpoint to a PSTN number
-                    createCallFromEndpoint(event, userName, baseAppUrl);
-
-                } else if (event.getProperty("to") != null) {
-                    // Case Incoming call from PSTN to a number associated to an Endpoint
-                    createCallToEndpoint(event, userName, baseAppUrl);
+                Map<String, CallEvents> callEventMap = userEventCallMap.get(userName);
```

Comment:
As I think about it, the variable `callEventMap` does actually make sense, because it is the event map for that specific call. Would it make more sense if `userEventCallMap` was renamed to something like `userEventMap`? The `userEventCallMap` maps users to calls.